### PR TITLE
Add PVC for Prometheus

### PR DESCRIPTION
**Pull request by mjudeikis**
This adds storage for Prometheus and Alertmanager

Fixes https://github.com/openshift/openshift-azure/issues/303

### CODE REVIEW

File: pkg/addons/data/StatefulSet.apps/openshift-metrics/prometheus.yaml [hunk 1]
```diff
@@ -178,9 +176,22 @@ spec:
...
+      resources:
+        requests:
+          storage: 10Gi
+  - metadata:
+      name: prometheus-data
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      resources:
+        requests:
+          storage: 100Gi
```
Comment (kargakis) [inline pkg/addons/data/StatefulSet.apps/openshift-metrics/prometheus.yaml]: nit: add newline

File: pkg/addons/data/StatefulSet.apps/openshift-metrics/prometheus.yaml [hunk 2]
```diff
@@ -178,9 +176,22 @@ spec:
...
+      resources:  
+        requests:
+          storage: 10Gi
+  - metadata:
+      name: prometheus-data
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      resources:
+        requests:
+          storage: 100Gi
```
[the remark this answers is not in the document]
Comment (mjudeikis) [reply pkg/addons/data/StatefulSet.apps/openshift-metrics/prometheus.yaml]: As we dont have any knowledge, we need to do error and trial here :/

[the remark this answers is not in the document]
Comment (jim-minter) [reply pkg/addons/data/StatefulSet.apps/openshift-metrics/prometheus.yaml]: see https://azure.microsoft.com/en-us/pricing/details/managed-disks/ - there's little point not asking for a disk size not listed here.  Suggest you go for 32Gi and 128Gi.

**Pull request merged**